xml/templates/message_templates/payment_or_refund_notification_html.tpl

```diff
@@ -147,7 +147,7 @@
     <tr>
       <td>
   <table style="border: 1px solid #999; margin: 1em 0em 1em; border-collapse: collapse; width:100%;">
-    {if $contributeMode eq 'direct' and !$isAmountzero}
+    {if $billingName || $address}
```

Comment:
these are assigned in the processCreditCard code. I would have liked to load them for the db but we don't retain an address per payment - only per contribution - probably we should